chromadb/logservice/grpc/client.py

```diff
@@ -0,0 +1,30 @@
+from typing import Sequence
+from uuid import UUID
+from chromadb.proto.chroma_pb2 import SubmitEmbeddingRecord
+from chromadb.proto.logservice_pb2 import PushLogsRequest
+from chromadb.proto.logservice_pb2_grpc import LogServiceStub
+import grpc
+
+
+class GrpcLogService:
```

Comment:
Suggestion to use the existing log abstractions or somehow rectify this with those.